.travis.yml

```diff
@@ -2,17 +2,16 @@ sudo: false
 
 language: go
 go:
-  - 1.7.x
```

Comment:
Why are we removing 1.7?

Reply:
I mention that in the commit message :)

>Using go dep requires support for the `sort` packages which was first
available in in go1.8

Reply:
I see. The idea is to release this with v12 right?

Reply:
Correct :)